Tests/WebTest/SitemapTest.php

```diff
@@ -87,14 +90,19 @@ public function getFormats()
                          <xhtml:link href="http://localhost/sitemap-aware?_locale=de" hreflang="de" rel="alternate"/>
                      </url>
                      <url>
+                         <loc>http://localhost/sitemap-aware-last-mod-date</loc>
+                         <lastmod>2016-07-06T00:00:00+02:00</lastmod>
```

Comment:
If will be very hard here, cause match the response string (it is a webtest). Should we Switch to a preg_match of strings instead of the complete strings.